BinaryEncoding.md

```diff
@@ -73,27 +73,37 @@ Strings referenced by the module (i.e. function names) are encoded as null-termi
 The following documents the current prototype format. This format is based on and supersedes the v8-native prototype format, originally in a [public design doc](https://docs.google.com/document/d/1-G11CnMA0My20KI9D7dBR6ZCPOBCRD0oCH6SHCPFGx0/edit?usp=sharing).
 
 ## High-level structure
-A module contains a sequence of sections, each identified by a *section identifier byte* whose encodings
-are listed below.
+A module contains a sequence of sections. Sections can in general be repeated, but some can occur only once or have dependant sections that must preceed them. Each section is identified by an immediate string. Sections whose identity is unknown to the WebAssembly implementation are ignored and this is supported by including the size in bytes for all sections. The encoding of all sections begins as follows:
```

Comment:
It would seem a significant and orthogonal change to give all the wasm sections a fixed order. It seems unnecessary from a source code point of view and might be frustrating even for the wast if it too required a fixed order, but can be worked by the tooling and in meta information, so if people could just confirm this decision I'm happy to note this here. But could this be split out into a follow up PR that someone else could champion? Or are you concerned to avoid the complexity of requiring a table lookup and want to settle this now?